src/app/views/discover/games/list/list.vue

```diff
@@ -41,24 +41,28 @@
 						</template>
 					</h1>
 
-					<template v-if="tag">
+					<p class="text-muted small">
+						{{ listDescription }}
+					</p>
+
+					<!-- <template v-if="tag">
```

Comment:
all of this is commented out?

Reply:
it has parts that had to do with date filters when they are set. I don't generate custom descriptions for these cases but since the functionality is there i didn't know what to do with it. What do you think? should we just remove this after all?

Reply:
yeah, remove